### PR TITLE
FeatureTableFX

### PR DESCRIPTION
**Pull request by SteffenHeu**
Hi Tomas,

this adds:
- Search for row types
- Column management per data type
 -> visibility is saved as a parameter
 -> TODO: Nested columns (like area bars, feature shape because they are not data types and not as modular as the rest)
- XIC / Spectra of selected row in the table

grafik

### CODE REVIEW

**Comment by robinschmid** [comment]
Some datatypes are SubColumnFactories and they produce the columns for the bar charts etc.

File: src/main/java/io/github/mzmine/datamodel/data/types/BestFeatureType.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package io.github.mzmine.datamodel.data.types;
```
Comment (tomas-pluskal) [inline src/main/java/io/github/mzmine/datamodel/data/types/BestFeatureType.java:1]: Please add the license header

**Comment by tomas-pluskal** [comment]
This looks nice, are you still working on it or is it ready to merge?

**Comment by SteffenHeu** [comment]
I would say it's ready to merge. I'll add the license later today.

**Pull request merged**